ckanext/switzerland/templates/footer_ogdch.html

```diff
@@ -2,14 +2,23 @@
 <section class="contribute inverted-container">
   <div class="container">
     <div class="row">
-      <div class="col-md-6 col-xs-12">
+      <div class="col-lg-3 col-md-4 col-sm-4 col-xs-12 ">
```

Comment:
I wasn't sure what combining the column classes like this did, so I looked it up and found out that the column tiers 'scale up', so if we want the same width for `sm` and `md`, we can leave out the `col-md-4` and only specify `col-sm-4`. :) (Hopefully this explains it clearer than I did: https://getbootstrap.com/docs/3.3/examples/grid/.)